### PR TITLE
Charlottetown things uno 1

### PR DESCRIPTION
**Pull request by jhthompson**
The most up to date Arduino code that I have worked on. There are some hardcoded values that identify the sensor, but are commented to indicate that.

### CODE REVIEW

File: arduino/main/main.ino [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
 #define DHTPIN 7     // what digital pin DHT temp sensor is connected to
 #define freqPlan TTN_FP_US915     // TTN_FP_EU868 or TTN_FP_US915 for European or US bandwidth
 
+// magic numbers for subtype of sensors, additional types can be added in the futures
+byte subtypes[10] = {
+  0x00, // indoor
+  0x01, // outdoor
+};
+
 // #define LCD_ATTACHED 1 // 1 if LCD screen connected, comment out if not
```
Comment (mregger) [inline arduino/main/main.ino]: Do we need to keep all the LCD stuff?

File: arduino/main/main.ino [hunk 2]
```diff
@@ -25,8 +36,8 @@
 #define debugSerial Serial
 
 // Uncomment whatever type you're using!
-#define DHTTYPE DHT11   // DHT 11
-//#define DHTTYPE DHT22   // DHT 22  (AM2302), AM2321
+//#define DHTTYPE DHT11   // DHT 11
```
Comment (mregger) [inline arduino/main/main.ino:32]: If this won't be used, might as well delete it

File: arduino/main/main.ino [hunk 3]
```diff
@@ -52,10 +63,10 @@ LiquidCrystal_I2C lcd(0x38, 16, 2);
 TheThingsNetwork ttn(loraSerial, debugSerial, freqPlan);
 
 // AppEUI and AppKey for The Things Network
-const char *appEui = "70B3D57ED000F6F6";
-const char *appKey = "ECDEFF8CAAA80B6A7EF0522B027DEFFC";
+const char *appEui = "70B3D57ED001712B";
```
Comment (mregger) [inline arduino/main/main.ino:55]: We need to hide this (otherwise any Arduino can have access to the DB

File: arduino/main/main.ino [hunk 4]
```diff
@@ -113,28 +133,34 @@ void loop() {
...
+  #ifdef DEBUG
+  debugSerial.print(F("Humidity: "));
+  debugSerial.print(h);
+  debugSerial.print(F("%  Temperature: "));
+  debugSerial.print(t);
+  debugSerial.print(F("°C "));
+  #endif
   //debugSerial.print(f);
   //debugSerial.print(F("°F  Heat index: "));
   //debugSerial.print(hic);
```
Comment (mregger) [inline arduino/main/main.ino]: Delete code that's commented out (if not useful)

**Review by mregger — CHANGES_REQUESTED** [review]
In general: do we need to keep all the LCD stuff, including the diagrams with it? It draws way too much power to be used in production. Maybe only for debugging?

File: arduino/main/main.ino (hunk 1 above)
Comment (jhthompson) [reply arduino/main/main.ino]: It can all be deleted, yeah

File: arduino/main/main.ino (hunk 2 above)
Comment (jhthompson) [reply arduino/main/main.ino:32]: Right now we have one device that is using the DHT11 and the other is using the DHT22 - I think we should keep that comment there for now

File: arduino/main/main.ino (hunk 3 above)
Comment (jhthompson) [reply arduino/main/main.ino:55]: Sounds good

File: arduino/main/main.ino (hunk 4 above)
Comment (jhthompson) [reply arduino/main/main.ino]: I'll do that

File: arduino/main/main.ino (hunk 2 above)
Comment (mregger) [reply arduino/main/main.ino:32]: Sounds good!

**Review by mregger — DISMISSED** [review]

File: arduino/main/main.ino (hunk 3 above)
Comment (jhthompson) [reply arduino/main/main.ino:55]: Committed a change to hide the app key, kept app EUI since it is same across all devices

**Pull request merged**